src/llmcompressor/modeling/config.py

```diff
@@ -0,0 +1,17 @@
+from pydantic import BaseModel, model_validator
+
+
+class CalibrationConfig(BaseModel):
+    moe_calibrate_all_experts: bool
+    moe_calibrate_gated_acts: bool
```

Comment:
nit: Consider renaming to something like `use_gated_outputs` since The name suggests it's about "calibrating
  gated activations" but it actually controls whether expert outputs contribute to the final result.